drive_to_gps/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>drive_to_gps</name>
+  <version>0.0.0</version>
+  <description>The drive_to_gps package</description>
+
+  <maintainer email="a8nanji@uwaterloo.ca">Azum Nanji</maintainer>
+
+  <license>TODO</license>
+
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>std_msgs</build_depend>
```

Comment:
personally if its more than one  specific type of dependency, i'll just use the `<depends>` tag.